src/components/new-simulation/simulation-outcome.tsx

```diff
@@ -28,9 +30,35 @@ export function SimulationOutcome({ control1, control2, outcomeHidden, setOutcom
   if (outcomeHidden) {
     return <button className="simulation-button result" onClick={() => setOutcomeHidden(false)} />;
   } else {
+    // Determine style and path of outcome border
+    // The border uses the same dashed style as the graph curves, so it must be an svg
+    // The border has rounded corners, so it's a combination of lines and arcs
+    const index = getAllExperiments(ac).findIndex(
+      controls => controls.option1 === control1 && controls.option2 === control2);
+    const borderClass = "line-style-" + index;
+    const strokeWidth = 3; // Keep in sync with .outcome-border stroke-width in simulation-outcome.scss
+    const borderHeight = 48; // Based on .outcome-area height in simulation-outcome.scss - margin space
+    const borderWidth = 392; // Based on .outcome-area width in simulation-outcome.scss - margin space
+    const borderRadius = 3;
+    const bWidth = borderWidth - (2 * borderRadius + strokeWidth);
+    const bHeight = borderHeight - (2 * borderRadius + strokeWidth);
+    const aPrefix = `${borderRadius},${borderRadius} 0 0 1`;
+    const top = `h ${bWidth} a ${aPrefix} ${borderRadius},${borderRadius}`;
+    const right = `v ${bHeight} a ${aPrefix} -${borderRadius},${borderRadius}`;
+    const bottom = `h -${bWidth} a ${aPrefix} -${borderRadius},-${borderRadius}`;
+    const left = `v -${bHeight} a ${aPrefix} ${borderRadius},-${borderRadius}`;
+    const path = `M ${borderRadius + strokeWidth / 2},${strokeWidth / 2} ${top} ${right} ${bottom} ${left} z`;
```

Comment:
OK, you don't want me to point this out _after_ you've done all this work, but `<rect>` in SVG supports `rx` and `ry` for rounding corners simply.  [docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/rect)

Reply:
😱 ... 🤦 
Wow. Thank you for pointing that out.